src/libs/ReportUtils.js

```diff
@@ -4043,6 +4044,21 @@ function getTaskAssigneeChatOnyxData(accountID, assigneeAccountID, taskReportID,
     };
 }
 
+/**
+ * Return the mention message of a list accounntID
+ * @param {Array<String>} accountIDs
+ * @returns {String}
+ */
+function getMentionMessage(accountIDs) {
+    const listMention = _.map(accountIDs, (accountID) => {
+        const personalDetail = lodashGet(allPersonalDetails, accountID);
+        const displayNameOrLogin =
+            LocalePhoneNumber.formatPhoneNumber(lodashGet(personalDetail, 'login', '')) || lodashGet(personalDetail, 'displayName', '') || Localize.translateLocal('common.hidden');
+        return `@${displayNameOrLogin}`;
+    });
+    return listMention.join(' and ');
```

Comment:
I think we should stick to the format with commas:

![image](https://github.com/Expensify/App/assets/2590174/4a7de7a0-1137-4d7f-bfd0-a9f23989d97a)